etc/scripts/run_uta_build_gene.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+if [ "$#" -lt 1 ]
+then
+    echo "error: too few arguments, you provided $#, 1 required"
+    echo "usage: run_uta_build_gene.sh <source_uta_v>"
+    exit 1
+fi
+source_uta_v=$1
+
+set -euxo pipefail
+
+loading_schema="uta_1_1"
```

Comment:
What is the significance of the hard-coded uta_1_1?

Reply:
The script makes a copy of the base (n -1) schema. For the new schema we need a name. We can either choose a default, or input one. Since the working schema is referenced in the uta config files it might be simplier to just use a default.